Engine/src/Application.bf

```diff
@@ -187,17 +216,51 @@ namespace SteelEngine
 				system.[Friend]PostUpdate();
 			}
 
+			for (var layer in _layerStack)
+				layer.OnUpdate();
+
+			_gameConsole.Update();
+
 			OnUpdate();
 		}
 
+		public void PushLayer(Layer layer)
+		{
+			_layerStack.PushLayer(layer);
+		}
+
+		public void PushOverlay(Layer layer)
+		{
+			_layerStack.PushOverlay(layer);
+		}
+
+		public static T GetInstance<T>() where T : Application
+		{
+			return (T) Instance;
+		}
+
+		struct PosColorVertex
```

Comment:
This struct in not referenced anywhere and shouldn't be defined in `Application.bf` file.

Reply:
Don't know how that got there. It is from the BGFX demo